helper/certutil/types.go

```diff
@@ -45,6 +45,16 @@ const (
 	TLSClient
 )
 
+//BlockType indicates the serialization format of the key
+type BlockType string
+
+//Well-known formats
+const (
+	PKCS1Block BlockType = "RSA PRIVATE KEY"
+	PKCS8Block BlockType = "PRIVATE KEY"
+	ECBlock    BlockType = "EC PRIVATE KEY"
+)
```

Comment:
Are you sure about that? From [the RFC](https://www.ietf.org/rfc/rfc3447.txt) I get the strong impression that the only key type for PKCS#1 is RSA, and that EC has [its own format](https://tools.ietf.org/html/rfc5915).